docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -190,12 +190,12 @@ Additionally, the following restrictions are applied to the schema:
 - The field `uniqueItems` cannot be set to true.
 - The field `additionalProperties` cannot be set to false.
 
-This feature is __alpha__ in v1.8 and may change in backward incompatible ways.
-Enable this feature using the `CustomResourceValidation` feature gate on
+This feature is __beta__ in v1.9.
```

Comment:
`from v1.9` instead of `in v1.9`.

Reply:
Split the difference? "This feature is beta beginning with v1.9" ?

Reply:
Yup, this sounds better! :+1: